scripts/update_092025/calculate_basic_pub_stats.py

```diff
@@ -0,0 +1,228 @@
+import pandas as pd
+from scipy import stats
+import numpy as np
+
+
+# --- Helper functions ---
+
+
+def calculate_descriptive_stats(data_series):
+    """Calculates descriptive stats: n, mean/SD, 95% CI for mean, median/IQR."""
+    s = pd.to_numeric(data_series, errors="coerce").dropna()
```

Comment:
Nit: Single-letter variable names are discouraged in Python. For readability:

```suggestion
    series = pd.to_numeric(data_series, errors="coerce").dropna()
```

There are a lot of one- and two-letter variable names in this file; it would be great to rename these for readability!